src/pkgcheck/checks/profiles.py

```diff
@@ -229,6 +254,17 @@ def __init__(self, *args, use_addon: addons.UseAddon, keywords_addon: addons.Key
             | use_addon.global_iuse_implicit
         )
 
+    def _report_unknown_atom(self, path, atom):
+        if not isinstance(atom, atom_cls):
+            atom = atom_cls(atom)
+        if matches := self.existence_repo.match(atom):
+            removal = max(x.time for x in matches)
+            removal = datetime.fromtimestamp(removal)
+            years = (self.today - removal).days / 365
```

Comment:
365 is not a very accurate approximation xP.

Reply:
It was good enough for `OutdatedBlocker` to not notice. But in true sense, it results in ~4 days since starts of Gentoo's life, which under rounding error is 0.01 of the year. I think we can slide it unless better solution found.

Reply:
A better solution was known for at least a few hundred years ;-).
```suggestion
            years = (self.today - removal).days / 365.2425
```